_posts/2020-03-20-status-update.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "20th March 2020: IAC Team Status Update"
+author: sheenaajay
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+  - facter
+  - testing
+  - pdk
+---
+
+The IAC team has been working on a lot of interesting things over the past week.
+Here's a brief overview of what we've been up to:
+
+## COVID-19
+Read [Puppet's blog: COVID-19](https://puppet.com/blog/covid-19-a-note-for-our-community/) for the official statement.
+Puppet provided remote working for all employees to support COVID-19.
+This precautionary measure has been taken in order to keep everyone healthy and limit the spread of the virus.
+
+The company supports employees by giving them all the essentials to work remotely and it also appreciates the fact that remote working requires everyone to adjust their working styles.
+It may not be smooth for everyone, but the company is doing its best to support all the employees during this challenging time.
+
+So far, everyone is well equipped to work remotely to their best.
+
+![Remote Working](/iac/assets/2020-03-20-status-update/remote_working.jpg)
```

Comment:
```suggestion
![Remote Working]({% link /assets/2020-03-20-status-update/remote_working.jpg %})
```

jekyll checks that the target exists and correctly renders it depending on the config. This makes it easier to test locally using `bundle exec jekyll build --strict --trace --verbose --baseurl=$(pwd)/_site --future`